eslint-local-rules.js

```
@@ -0,0 +1,353 @@
+          missingLabel: 'Add an "id" attribute to this {{tagName}} field and associate it with a label.'
+        }
+      },
+      create(context) {
+        return {
+          Program() {
+            const sourceCode = context.sourceCode;
+            const text = sourceCode.getText();
+            const controlRegex = /<(input|select|textarea)\b([^>]*)>/gi;
+            let match;
+
+            while ((match = controlRegex.exec(text)) !== null) {
```

Biome (2.5.0)

[error] The assignment should not be in an expression.

(lint/suspicious/noAssignInExpressions)